python/src/gen3authz/client/arborist/base.py

```diff
@@ -694,12 +741,19 @@ async def list_resources_for_user(self, username):
         return response.json["resources"]
 
     @maybe_sync
-    async def grant_user_policy(self, username, policy_id):
+    async def grant_user_policy(self, username, policy_id, expires_at=None):
         """
-        MUST be user name, and not serial user ID
+        Grant a policy to a user
+
+        Args:
+            username (str): MUST be user's username, and not serial user ID
+            policy_id (str): Arborist policy id
+            expires_at (datetime.datetime): when the policy should expire (in UTC)
```

Comment:
we use/pass around POSIX timestamps in most of the Gen3 stack, since it's arborist that needs UTC, can we handle all that in this function? e.g. can we pass in an integer POSIX timestamp here and do the https://github.com/uc-cdis/fence/pull/986/files#diff-5824dff91d0ccd46626b8d7fbb03bcc2eb6882c9c038139c48ccfd8c39082d70R1670 stuff here? so fence still just handles/passes ints around?